### PR TITLE
removing smiley

### PR DESCRIPTION
**Pull request by skarode96**
please review this PR

### CODE REVIEW

File: experiment.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ else {
 }
 */
 const timesToRepeat = 50;
-const character = '🥰😇'
```
Comment (Navi97) [inline experiment.js:28]: please add one more sad simely

File: experiment.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ else {
...
+const character = '😇'
```
Comment (Navi97) [inline experiment.js]: please remove this smiley

File: experiment.js (hunk 1 above)
Comment (skarode96) [reply experiment.js:28]: done

File: experiment.js (hunk 2 above)
Comment (skarode96) [reply experiment.js]: done

**Review by Navi97 — APPROVED** [review]
looks good

**Pull request merged**